sdk/nodejs/computeCluster.ts

```diff
@@ -63,11 +63,11 @@ import * as utilities from "./utilities";
  * }));
  * const hostsHost: Output<vsphere.GETHOSTResult>[] = [];
  * for (let i = 0; i < hosts.length; i++) {
- *     hostsHost.push(pulumi.output(vsphere.getHost({
- *         datacenterId: dc.apply(dc => dc.id),
+ *     hostsHost.push(vsphere.getHost);
+ * %!(EXTRA string=dc.apply(dc => vsphere.getHost({
```

Comment:
@pgavlin Not really sure what this means, I have the latest version of `tf2pulumi` on my path and it generated this.

Reply:
Oh, interesting. This is a bug. Can you file an issue in https://github.com/pulumi/tf2pulumi?

Reply:
https://github.com/pulumi/tf2pulumi/issues/83  Going to merge this as is.